docs/resources/migration.md

```diff
@@ -0,0 +1,79 @@
+**Dependencies** - Harmony doesn't allow relative import statements between components. Each Bit component always has a local, compiled module in node_modules - use them for all Bit component import statements in your code.  
+**Environments** - Harmony features a new approach for compilers/testers/etc called environments. Take the base environment per your component's stack (react, node, react-native) and customize it to fit any specific configuration requirement.  
+**Configuration** - In harmony config is managed in a new file format - `workspace.jsonc` and not in your `package.json`.  
+**Module names** - In harmony we removed the @bit prefix, and instead components are now pre-fixed by @<account_name>. Your import statements are likely to need updating.  
+
+
+### Moving from Legacy Bit to Harmony
+
+Now that we've outlined the above, there are essentially 4 major steps required to migrate components over from using legacy versions of Bit (14-) to Harmony.
+
+1. Reset your bit workspace to be a Harmony workspace, or if you would like to keep your legacy workspace
+1. Configure your Harmony [Component Development Environment](/building-with-bit/environments)(s) with the compiler, tester, etc configurations (most likely, with those you used when using legacy Bit). If you don't have any complex configurations, it's worth starting with one of our base Environments, which may just do the job out of the box, and if you need any customizations then [create your own custom environment](/building-with-bit/environments#customizing-environments) based on the base Environment.
+Understanding and configuring (where required) Environments is the central learning curve when using Bit - once the environment is configured correctly, the component life-cycle essentially takes care of itself.
+1. Copy over the component code, making sure all components files are contained under a single directory, including the entry file into your component. Then track the components with Bit, in much the same way you would with previous Bit versions.
+1. Configure your workspace, specifically the variant/s section to create rules e.g. for mapping the relevant environment (built-in or customized) to your components
+
+Those are the major high-level steps that are required for the migration, which we'll elaborate on below. 
+
+In general Harmony works differently under the hood to legacy Bit versions, so some tinkering may be required with configurations such as compiler configs or dependency resolution rules. But in the main, issues you may come across are likely to be documented either in the general [Harmony docs](https://harmony-docs.bit.dev/), on our [community slack](https://join.slack.com/t/bit-dev-community/shared_invite/zt-o2tim18y-UzwOCFdTafmFKEqm2tXE4w) or on [github](https://github.com/teambit/bit/issues).
```

Comment:
1) Not sure what's irrelevant about this. It's the exact list of steps needed to migrate. Care to elaborate?
2) What does it go against?
3) The workspace resetting / cloning is actually the first item in the list. Looks like I just left it in the middle though, so the second part (cloning and having it in a separate workspace) isnt there. But even if they clone, they still need to know how to reset the cloned repo as it will have legacy bit intertwined with it

In general this whole comment is full of general statements - please try to be more concrete in the feedback, it helps promote conversation and especially in understanding what the actual feedback is